tests/gorepo/run.go

```diff
@@ -1029,7 +1047,7 @@ func (t *test) errorCheck(outStr string, fullshort ...string) (err error) {
 
 func (t *test) updateErrors(out string, file string) {
 	// Read in source file.
-	src, err := os.ReadFile(file)
+	src, err := ioutil.ReadFile(file)
```

Comment:
Was it intentional to switch back to `ioutil` on this line and others like it?

Reply:
Nope, that was an unintended effect of me cherry-picking some changes from the original generics branch. Fixed.